libs/libsystem/malloc.c

```diff
@@ -0,0 +1,108 @@
+#include "malloc.h"
+#include "syscalls.h"
+#include "string.h"
+

```

Comment:
Add definition of `__alloc_new_block` here

Reply:
on tyt i est'